cmd/root.go

```diff
@@ -59,17 +59,17 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "",
-		"config file (default is $HOME/.deck.yaml)")
+		"Config file (default is $HOME/.deck.yaml).")
 
 	rootCmd.PersistentFlags().String("kong-addr", "http://localhost:8001",
-		"HTTP Address of Kong's Admin API.\n"+
-			"This value can also be set using DECK_KONG_ADDR\n"+
+		"HTTP address of Kong's Admin API.\n"+
+			"This value can also be set using the DECK_KONG_ADDR\n"+
```

Comment:
```suggestion
			"This value can also be set using the environment variable DECK_KONG_ADDR\n"+
```
(I think that's correct, but might want to verify)